java/app/src/main/java/com/example/chattutorial/MainActivity.java

```diff
@@ -33,9 +34,11 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // setup the client using the example API key
         // normally you would call init in your Application class and not the activity
-        ChatClient client = new ChatClient.Builder("b67pax5b2wdq", getApplicationContext()).logLevel(
+        Chat chat = new Chat.Builder("b67pax5b2wdq", getApplicationContext()).logLevel(
                 ChatLogLevel.ALL).build();
 
+        ChatClient client = chat.getClient();
```

Comment:
How do you change things on the ChatClient instance?

* Is everything that's configurable on the ChatClient also configurable on the Chat object?
* Alternatively can you pass a ChatClient object to Chat in the constructor?

Reply:
- `Chat.Builder` contains all options of `ChatClient.Builder`: https://github.com/GetStream/stream-chat-android/blob/4b6cbb85f0b70ba0690cd0fb053e8bc944bbb9b2/library/src/main/java/com/getstream/sdk/chat/Chat.java#L48
 - No, because all options already available `Chat.Builder`. Also `Builders` are required, because they create static references: `Chat.instance` and `ChatClient.instance`. And these refrences are used inside SDKs.